ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -64,6 +64,11 @@ timeout -s 9 175m ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF |& sed -e 's/.*
 
 set -xeuo pipefail
 
+# Some Packet images have a file /usr/config left from the provisioning phase.
+# The problem is that sos expects it to be a directory. Since we don't care
+# about the Packet provisioner, remove the file if it's present.
+rm -f /usr/config || true
```

Comment:
I think the original form propose by @dtantsur was preferrable, ie: 
```
test -f /usr/config && rm -f /usr/config || true
```

Reply:
Fixed!